### DISCUSSION TITLE
FCOS moving iptables to the nft backend

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [okd-project/okd#613] [General]
This thread is to collate all the information we have on handling FCOS's move of iptables to the nft backend, including the game plan for OKD to support it.

https://github.com/coreos/fedora-coreos-tracker/issues/676

**Comment by vrutkovs** [comment]
I don't think there's something requires doing - all NFT changes are handled by SDNs. Since RHEL8 (and thus RHCOS) are using NFT by default we'd get NFT support for free
